scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -144,10 +144,18 @@ package object dialects {
     .withAllowStarWildcardImport(true)
     .withAllowProcedureSyntax(false)
     .withAllowDoWhile(false)
-    .withAllowPlusMinusUnderscoreAsPlaceholder(true)
     .withUseInfixTypePrecedence(true)
     .withAllowInfixOperatorAfterNL(true)
 
+  implicit val Scala31 = Scala30.duplicate
+
+  implicit val Scala32 = Scala31
```

Comment:
Actually, it seems it might not be supported by Scala 3.2 yet :thinking: The docs require some updates, I contacted the maintainers.

Reply:
it was actually supported earlier, in [3.0.1](https://github.com/lampepfl/dotty/releases/tag/3.0.1). i kept it under 3.2, though, following the documentation.